files/build_templates/qos_config.j2

```diff
@@ -49,6 +49,11 @@
 
 {%- set require_global_dscp_to_tc_map = true -%}
 
+{%- set vars = {'backplane_port_exist': false} %}
+{% for port in PORT_ACTIVE if port.startswith('Ethernet-BP')%}
```

Comment:
There are other multi-asic platforms with port names starting with 'Ethernet-BP' and this change could affect those

Reply:
Added a flag backplane_port_no_ecn_marking for it.